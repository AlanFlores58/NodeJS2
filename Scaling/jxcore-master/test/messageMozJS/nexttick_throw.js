// Copyright & License details are available under JXCORE_LICENSE file


var common = require('../common');
var assert = require('assert');

process.nextTick(function() {
  process.nextTick(function() {
    process.nextTick(function() {
      process.nextTick(function() {
        undefined_reference_error_maker;
      });
    });
  });
});