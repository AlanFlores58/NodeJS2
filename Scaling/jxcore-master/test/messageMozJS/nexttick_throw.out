ReferenceError: undefined_reference_error_maker is not defined (*test*messageMozJS*nexttick_throw.js *:*)
    at  (*test*messageMozJS*nexttick_throw.js:*:*)
    at _tickCallback (node.js:*:*)
    at Module.runMain (module.js:*:*)
    at startup (node.js:*:*)
    at node.js:*:*
